src/XMakeBuildEngine/UnitTests/BackEnd/TargetUpToDateChecker_Tests.cs

```diff
@@ -903,6 +909,122 @@ bool includeOutput2
                 if (File.Exists(output2)) File.Delete(output2);
             }
         }
+
+        private static readonly DateTime Old = new DateTime(2000, 1, 1);
+        private static readonly DateTime Middle = new DateTime(2001, 1, 1);
+        private static readonly DateTime New = new DateTime(2002, 1, 1);
+
+        [Fact(Skip = "Creating a symlink on Windows requires elevation.")]
```

Comment:
Well this sucks :(
I wonder if we could run elevated on CI builds and then conditionally run the tests if we detect sufficient access.